tests/foreman/api/test_discoveredhost.py

```diff
@@ -117,35 +118,27 @@ def setUpClass(cls):
             "/var/lib/tftpboot/pxelinux.cfg/default"
         )
         ssh.command(
-            "sed -ie '/APPEND initrd/s/$/ fdi.countdown=1/' "
+            "sed -ie '/APPEND initrd/s/$/ fdi.countdown=1 fdi.ssh=1 fdi.rootpw=changeme/' "
```

Comment:
I didn't bother parametrizing the discovered host root password, as it doesn't really matter - i implemented this in order to gain access to the VM in case I decide to implement deeper assertions.
If you believe it shouldn't be here until it is really used, i can remove it.